frameworks/g-cloud-7/manifests/edit_service_as_admin.yml

```diff
@@ -47,195 +44,3 @@
-    - datacentreTier
-    - dataBackupRecovery
-    - dataExtractionRemoval
--
-  name: Data-in-transit protection
-  editable: True
-  questions:
-    - dataProtectionBetweenUserAndService
-    - dataProtectionWithinService
-    - dataProtectionBetweenServices
--
-  name: Asset protection and resilience
-  editable: True
-  questions:
-    - datacentreLocations
-    - dataManagementLocations
-    - legalJurisdiction
-    - datacentreProtectionDisclosure
-    - dataAtRestProtections
-    - dataSecureDeletion
-    - dataStorageMediaDisposal
-    - dataSecureEquipmentDisposal
-    - dataRedundantEquipmentAccountsRevoked
-    - serviceAvailabilityPercentage
--
-  name: Separation between consumers
-  editable: True
-  questions:
-    - cloudDeploymentModel
-    - otherConsumers
-    - servicesSeparation
-    - servicesManagementSeparation
--
-  name: Governance
-  editable: True
-  questions:
-    - governanceFramework
--
-  name: Configuration and change management
-  editable: True
-  questions:
-    - configurationTracking
-    - changeImpactAssessment
--
-  name: Vulnerability management
-  editable: True
-  questions:
-    - vulnerabilityAssessment
-    - vulnerabilityMonitoring
-    - vulnerabilityMitigationPrioritisation
-    - vulnerabilityTracking
-    - vulnerabilityTimescales
--
-  name: Events and Incidents
-  editable: True
-  questions:
-    - eventMonitoring
-    - incidentManagementProcess
-    - incidentManagementReporting
-    - incidentDefinitionPublished
--
-  name: Personnel security
-  editable: True
-  questions:
-    - personnelSecurityChecks
--
-  name: Secure development
-  editable: True
-  questions:
-    - secureDevelopment
-    - secureDesign
-    - secureConfigurationManagement
--
-  name: Supply-chain security
-  editable: True
-  questions:
-    - thirdPartyDataSharingInformation
-    - thirdPartySecurityRequirements
-    - thirdPartyRiskAssessment
-    - thirdPartyComplianceMonitoring
-    - hardwareSoftwareVerification
--
-  name: Authentication of consumers
-  editable: True
-  questions:
-    - userAuthenticateManagement
-    - userAuthenticateSupport
--
-  name: Separation and access control within management interfaces
-  editable: True
-  questions:
-    - userAccessControlManagement
-    - restrictAdministratorPermissions
-    - managementInterfaceProtection
--
-  name: Identity and authentication
-  editable: True
-  questions:
-    - identityAuthenticationControls
--
-  name: External interface protection
-  editable: True
-  questions:
-    - onboardingGuidance
-    - interconnectionMethods
--
-  name: Secure service administration
-  editable: True
-  questions:
-    - serviceManagementModel
--
-  name: Audit information provision to consumers
-  editable: True
-  questions:
-    - auditInformationProvided
--
-  name: Secure use of the service by the customer
-  editable: True
-  questions:
-    - deviceAccessMethod
-    - serviceConfigurationGuidance
-    - trainingProvided
--
-  name: Documents
```

Comment:
I think we want to keep the document editing?  Nicole already has it and it works.